### PR TITLE
Added whats next section to application overview

### PR DESCRIPTION
**Pull request by rohitkrai03**
## Fixes 

- https://issues.redhat.com/browse/HAC-2686

## Description

- Added what's next section to application overview page.
- Using random help topic id for all learn more links. Will be updating it with correct ones when docs create them.
- Need to add some unit tests.

## Type of change

- [x] Feature

## Screen shots / Gifs for design review 

https://user-images.githubusercontent.com/6041994/215546704-16e99091-d373-4bc9-b613-898ef288b9d6.mov

## Browser conformance: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: src/components/ApplicationDetails/tabs/ApplicationOverviewTab.tsx [hunk 1]
```diff
@@ -1,16 +1,67 @@
...
+    {
+      title: 'Install our GitHub app',
+      description: 'Install the GitHub app to monitor your work from a commit to deployment.',
+      icon: githubAppIcon,
+      cta: {
+        label: 'Start the flow',
+        href: 'https://github.com/apps/appstudio-staging-ci',
+        external: true,
+      },
+      helpId: 'add-component',
```
Comment (sahil143) [inline src/components/ApplicationDetails/tabs/ApplicationOverviewTab.tsx:35]: Do we need to hide the learn more link for ids/section that don't have content?

**Review by sahil143 — COMMENTED** [review]
Looks good overall!
The layout seems a bit broken when the help topic panel is open

image

File: src/components/ApplicationDetails/tabs/ApplicationOverviewTab.tsx (hunk 1 above)
Comment (rohitkrai03) [reply src/components/ApplicationDetails/tabs/ApplicationOverviewTab.tsx:35]: We'll be getting the learn more ids for all the whats next items from docs team. I've used `add-component` for now just to show how it looks.

**Review by sahil143 — COMMENTED** [review]
@rohitkrai03 Remove icons that are not needed in the What's next section. Jeff's PR https://github.com/openshift/hac-dev/pull/372 is already importing other icons and we will end up with duplicate icons.

**Comment by rohitkrai03** [comment]
> @rohitkrai03 Remove icons that are not needed in the What's next section. Jeff's PR #372 is already importing other icons and we will end up with duplicate icons.

Removed all unused illustrations.

**Comment by rohitkrai03** [comment]
> Looks good overall! The layout seems a bit broken when the help topic panel is open
> 
> image

@sahil143 Fixed the layout issue when side panel is open. Now the CTA buttons will be consistent.

Screenshot 2023-02-01 at 2 50 31 PM

**Review by sahil143 — APPROVED** [review]
/lgtm

**Pull request merged**